### PR TITLE
Temporarily specify us-east-1 region for HCP tests to stop bleeding w…

### PR DESCRIPTION
**Pull request by svetsa-rh**
Until changes to us-west-2 region are successfully rolled and complete, moving tests to us-east-1 
Yesterday I ran a test in us-east-1 and the cluster was successfully created.
https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/origin-ci-test/pr-logs/pull/openshift_release/45058/rehearse-45058-pull-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa_hcp-4.15-nightly-x86-perfscale-rosa-hcp-1zone-data-path-2nodes-periodic/1719151368830193664/artifacts/perfscale-rosa-hcp-1zone-data-path-2nodes-periodic/rosa-cluster-provision/build-log.txt 

Jira for reference: https://issues.redhat.com/browse/PERFSCALE-2606

### CODE REVIEW

**Comment by svetsa-rh** [comment]
/pj-rehearse ack

I verified  us-east-1 works yesterday [https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/origin-ci-test/pr-logs/pull/openshift_rele[…]s-periodic/rosa-cluster-provision/build-log.txt](https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/origin-ci-test/pr-logs/pull/openshift_release/45058/rehearse-45058-pull-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa_hcp-4.15-nightly-x86-perfscale-rosa-hcp-1zone-data-path-2nodes-periodic/1719151368830193664/artifacts/perfscale-rosa-hcp-1zone-data-path-2nodes-periodic/rosa-cluster-provision/build-log.txt)  and cluster was created successfully.

File: ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__rosa_hcp-4.13-nightly-x86.yaml [hunk 1]
```diff
@@ -93,6 +96,7 @@ tests:
       HOSTED_CP: "true"
       MULTI_AZ: "false"
       OPENSHIFT_VERSION: "4.15"
+      REGION: us-east-1
```
Comment (paigerube14) [inline ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__rosa_hcp-4.13-nightly-x86.yaml:99]: i saw this one was in just a normal rosa cluster, do we need to change the rosa ones too

Comment (svetsa-rh) [reply ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__rosa_hcp-4.13-nightly-x86.yaml:99]: No, the region changes are needed only for HCP clusters.
This reminds me, you might want to change Chaos HCP tests too.

**Comment by svetsa-rh** [comment]
We will close this PR now as `us-west-2` is back and we were able to successfully run a sanity test there https://prow.ci.openshift.org/view/gs/origin-ci-test/logs/periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa_hcp-4.14-nightly-x86-perfscale-rosa-hcp-3zones-node-density-heavy-24nodes-periodic/1719392294437654528

**Comment by svetsa-rh** [comment]
OOPs.. mean to close

**Pull request closed without merging**